src/components/App.js

```diff
@@ -1,17 +1,40 @@
+/* eslint-disable react/no-access-state-in-setstate */
```

Comment:
- [ ] Please don't disbale eslint. Please, let's try to resolve the errors.

- [ ] Please let's fix the bug in the `+/-` operator. It's currently outputing the negative square of a number. eg. pressing `8` and `+/-` gives `-64`.
- [ ] Decimal point is also not working. Please let's fix it.

Reply:
thanks for the review I have fixed the buttons not working. I think they work fine now. but I'm not able to find solution for the linter error , it is saying to use callbacks and I can't seem to find a way to do this simply with no hooks as I have not yet learnt hooks. if you have working code for the same implementation I used that would be very helpful. I have spent hours and was unable to resolve.